SonarQube.Bootstrapper/BuildAgentUpdater.cs

```diff
@@ -99,6 +117,81 @@ public bool CheckBootstrapperApiVersion(string versionFilePath, Version bootstra
             return false;
         }
 
+
+        public void InjectLoaderTargets(ILogger logger)
+        {
+            WarnOnGlobalTargetsFile(logger);
+            InternalCopyTargetsFile(logger);
+        }
+
+        private static void InternalCopyTargetsFile(ILogger logger)
+        {
+            string sourceTargetsPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), LoaderTargetsName);
```

Comment:
MSDN suggests using typeof(X).Assembly.Location instead of Assembly.GetExecutingAssembly().Location: https://msdn.microsoft.com/en-us/library/system.reflection.assembly.getexecutingassembly(v=vs.110).aspx

... although I am sure we are using GetExecutingAssembly elsewhere in the code.